### ISSUE TITLE
Create Admin Settings UI

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#8]
Mockup components for 'list view' of users and ability to edit users. 

AC:
- [x] Admin views a list view of users 
- [x] Admin views a form control for editing users
- [x] Mockup uses https://designsystem.digital.gov/

DoD
- [x] Design has been shared with the PO and team for feedback

**Comment by lfrohlich** [comment]
Should this be  a task under a user story as opposed to its own issue?

**Comment by shubhi-raft** [comment]
Could be. However, something to think about is - if it is included within a user story then the estimated story point will increase and it may bigger than we are hoping. 

Having a separate issue but tied/reference  to the bigger issue may be easier way to keep track.

**Comment by shubhi-raft** [comment]
> Should this be a task under a user story as opposed to its own issue?

@lfrohlich it could be. However, something to think about is - if it is included within a user story then the estimated story point will increase and it may bigger than we are hoping. furthermore, design team can work in parallel w/ dev team without formalizing 'handoffs'.

Having a separate issue but tied/reference to the bigger issue may be easier way to keep track. this issue blocks the other user story so it is tied to the other user story.

**Comment by lfrohlich** [comment]
Thanks @shubhi-raft . Makes sense to keep separate.

**Comment by lfrohlich** [comment]
Quick preview at Sprint Review 8/4

**Comment by cmajel** [comment]
@reitermb @shubhi-raft added some additional comments on the updated admin figma wires:

https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/Login-%2F-Admin-Mockups-For-Feedback?node-id=0%3A1

**Comment by shubhi-raft** [comment]
@matthewjdoty FYI

**Comment by reitermb** [comment]
@cmajel @lfrohlich It's ready now. I believe the most recent version (same link as above) has addressed all the feedback we got, but let me know if anything was missed!

Re: The tone of the placeholders on landing pages, those are definitely just for the purposes of initial dev implementation/testing. We anticipate another pass soon that puts things into a far more user-centric context there as we dig into #58

**Comment by cmajel** [comment]
🎉 Awesome looks good to me! 

Do we want to create a placeholder issue to revisit these flows and their content after additional research?

**Comment by lfrohlich** [comment]
Looks good to me! Thanks
